tests/scanner/scanners/config_validator_util/cv_data_converter_test.py

```diff
@@ -90,6 +90,18 @@ def test_generate_ancestry_path_for_folder(self):
         actual_path = cv_data_converter.generate_ancestry_path(full_name)
         self.assertEqual(expected_path, actual_path)
 
+    def test_generate_ancestors_for_org(self):
+        full_name = 'organization/1234567890/project/test-project-123/firewall/1234567890123456789/'
+        expected_ancestors = ['organization/1234567890', 'project/test-project-123']
+        actual_ancestors = cv_data_converter.generate_ancestors(full_name)
+        self.assertListEqual(expected_ancestors, actual_ancestors)
+
+    def test_generate_ancestors_for_folder(self):
+        full_name = 'folder/folder-1/project/project-2/lien/p123/'
```

Comment:
nit: folders are always a numeric ID, projects as given by CAI use project number for ancestry

Reply:
That's a good catch, thanks. Perhaps instead of hacking this together I should just make the change to the Forseti inventory/model tables to store the CAI ancestors. Since Forseti was already using some custom code to determine the ancestry path, I thought it wouldn't be terrible to just do the same for the ancestors. Let me try going that route instead; unless you have other ideas.

Reply:
This is only test code, so updating the values here to numeric would be more of a best practice.  Overall this change makes sense until things get fixed up in the Forseti inventory model/tables (DB changes tend to be pretty heavyweight).

Reply:
I have updated the folders to use the numeric ID and left the projects to use the ID instead of the numeric ID since that is what Forseti will be using. Org and folders will pass the numeric ID to CV, and the project ID.